src/algorithms/tracking/CKFTrackingFunction.cc

```diff
@@ -38,8 +38,13 @@ namespace eicrecon{
   using Navigator  = Acts::Navigator;
   using Propagator = Acts::Propagator<Stepper, Navigator>;
 
+#if Acts_VERSION_MAJOR >= 36
+  using CKF =
+      Acts::CombinatorialKalmanFilter<Propagator, ActsExamples::TrackContainer>;
+#else
   using CKF =
       Acts::CombinatorialKalmanFilter<Propagator, Acts::VectorMultiTrajectory>;
+#endif
 
   using TrackContainer =
       Acts::TrackContainer<Acts::VectorTrackContainer,
```

Comment:
I think `TrackContainer` is now unused for Acts_VERSION_MAJOR >= 36.

Reply:
Likewise, `#include <Acts/EventData/VectorMultiTrajectory.hpp>` is probably unnecessary for Acts_VERSION_MAJOR >= 36.